spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -78,4 +78,28 @@
       expect { subject }.not_to change { Target.count }
     end
   end
+
+  context 'when the user is not authenticated' do
+    subject { post api_v1_targets_url, params: target_params, as: :json }
+    let(:target_params) do
+      {
+        target: attributes_for(:target).merge(topic_id: topic.id)
+      }
+    end
+
+    it 'returns the error messages as a json' do
+      subject
+      error_msg = I18n.t('devise.failure.unauthenticated')
```

Comment:
As the message is not so long, you can avoid using a variable, and just call `I18n.t('devise.failure.unauthenticated')` inside the expect.